wampy/session.py

```diff
@@ -161,7 +161,7 @@ def connection_handler():
             while True:
                 try:
                     frame = connection.receive()
-                    if frame:
+                    if frame != None:
```

Comment:
i don't understand why you need this - can you explain?

Reply:
You are right, this should  not be necessary given the modification above